components/ContributeProject.tsx

```diff
@@ -0,0 +1,134 @@
+import Image from 'next/image';
+import React, { useState } from 'react';
+import { GFIProject, GitHubColors, GithubIssueAssignees } from '../util';
+import ELink from './ELink';
+
+
+interface ContributeProjectProps {
+  project: GFIProject,
+  githubColors: GitHubColors
+}
+
+export default function ContributeProject({
+  project,
+  githubColors,
+}: ContributeProjectProps): JSX.Element {
+  const {issues} = project;
+  const proppedProject = project.project;
+  const {
+    name,
+    description,
+    repo,
+    lang,
+    topics,
+    image,
+    alt,
+  } = proppedProject;
```

Comment:
This can be condensed, right?

```suggestion
  const {
    name,
    description,
    repo,
    lang,
    topics,
    image,
    alt,
  } = project.project;
```